### PR TITLE
add fontawesome via npm

### PR DESCRIPTION
**Pull request by creativenobu**
Solves issue #47

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -4,13 +4,11 @@
...
     <link rel="stylesheet" href="reset.css" type="text/css" media="screen" />
     <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Cantarell" />
-    <!-- TODO(#47): get fontawesome from npm -->
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
     <link rel="stylesheet" href="main.css" type="text/css" media="screen" />
   </head>
   <body>
     <div class="header">
-      <h1>Schedule</h1>
+      <h1>Schedule  <span class="fab fa-twitch"></span></h1>
```
Comment (rexim) [inline html/index.html]: This is completely outside of the scope of #47. Please remove this icon.

**Comment by creativenobu** [comment]
> Can we use only CSS part of the FontAwesome and still install it from npm?

I also don't like the JS approach, I guess I can include the FontAwesome icons using their SCSS. That way styling and logic are separate.

File: html/index.html (hunk 1 above)
Comment (creativenobu) [reply html/index.html]: Ah yes, I forgot to remove. That was a check to see if it worked or not. Will update

**Comment by rexim** [comment]
@creativenobu I'd really appreciate if you took a look into that, yes! If it takes too much effort to implement, please, don't waste your time on it. Just post your findings in this PR and I will see what we can do about it. Research is a contribution too. Thank you so much for your time!

**Comment by creativenobu** [comment]
So I've moved it to import FontAwesome via sass, seems like it works. I left a test case in main.scss (commented) to add FontAwesome icons in existing classes, if needed. 

I had to load the fontawesome path on the sass command, so I added SASS_FLAGS in the Makefile, that will include fontawesome scss and webfonts to the app.

I followed directions [here ](https://fontawesome.com/how-to-use/on-the-web/using-with/sass) and found the [sass cli ](https://sass-lang.com/documentation/cli/dart-sass) that had that option (fortunately?) 😄

**Comment by rexim** [comment]
@creativenobu this is waay better! Thank you so much for your contribution! :+1:
